python/servo/build_commands.py

```diff
@@ -433,29 +433,37 @@ def package_gstreamer_dlls(servo_exe_dir: str, target: str):
     return not missing
 
 
-def package_msvc_dlls(servo_exe_dir, target):
-    msvc_deps = [
-        "msvcp140.dll",
-        "vcruntime140.dll",
-    ]
-    if "aarch64" not in target != "aarch64":
-        msvc_deps += ["api-ms-win-crt-runtime-l1-1-0.dll"]
+def package_msvc_dlls(servo_exe_dir: str, target: str):
+    def copy_file(dll_path: Optional[str]) -> bool:
+        if not dll_path or not os.path.exists(dll_path):
+            print(f"WARNING: Could not find DLL at {dll_path}", file=sys.stderr)
+            return False
+        servo_dir_dll = path.join(servo_exe_dir, os.path.basename(dll_path))
+        # Avoid permission denied error when overwriting DLLs.
+        if os.path.isfile(servo_dir_dll):
+            os.chmod(servo_dir_dll, stat.S_IWUSR)
+        print(f"    • Copying {dll_path}")
+        shutil.copy(dll_path, servo_exe_dir)
+        return True
 
-    missing = []
-    redist_dirs = find_msvc_redist_dirs(target)
-    for msvc_dll in msvc_deps:
-        for dll_dir in redist_dirs:
-            dll = path.join(dll_dir, msvc_dll)
-            servo_dir_dll = path.join(servo_exe_dir, msvc_dll)
-            if os.path.isfile(dll):
-                if os.path.isfile(servo_dir_dll):
-                    # avoid permission denied error when overwrite dll in servo build directory
-                    os.chmod(servo_dir_dll, stat.S_IWUSR)
-                shutil.copy(dll, servo_exe_dir)
-                break
-        else:
-            missing += [msvc_dll]
+    vs_platform = {
+        "x86_64": "x64",
+        "i686": "x86",
+        "aarch64": "arm64",
+    }[target.split('-')[0]]
 
-    for msvc_dll in missing:
-        print(f"Could not find DLL dependency: {msvc_dll}")
-    return not missing
+    for msvc_redist_dir in find_msvc_redist_dirs(vs_platform):
+        if copy_file(os.path.join(msvc_redist_dir, "msvcp140.dll")) and \
+           copy_file(os.path.join(msvc_redist_dir, "vcruntime140.dll")):
+            break
+
+    if "aarch64" not in target != "aarch64":
```

Comment:
Is this conditional correct?

Reply:
I think it isn't. Additionally, I suspect this is left over from the UWP build which didn't use the Windows SDK in the same way. I'll just go ahead and remove it.